server/config/config.json

```diff
@@ -2,7 +2,7 @@
   "development": {
     "username": "postgres",
     "password": null,
-    "database": "docman-dev",
+    "database": "docmandev",
```

Comment:
I think it's fine the way it is given that she might be using a local DB for development. What do you think @enahomurphy